File: faculdade/semestre-01/programacao-de-computadores/exercicios/aula-02-atribuicao-entrada-e-saida/exerc02.py
#Escreva um programa em python que solicite ao usuario o salario atual e mostre o salário acrescido de 5% de comissão

salario_atual = float(input("Digite o seu salário atual: "))
comissão = salario_atual + (salario_atual*0.05)

print("O seu salário com a comsissão será: ",comissão)

File: Java/exercicios-diversos/exercicios-diversos/src/exercicios/exerc02.java
//Escreva um programa que solicite ao usuario o salario atual e mostre o salário acrescido de 5% de comissão

package exercicios;

import java.util.Scanner;

public class exerc02 {
    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite o seu salário atual: ");
        double salario = sc.nextInt();
        double comissao = salario+(salario*0.05);
        System.out.println("Seu salário com a comissão fica em: R$"+comissao);
    }
}
